index.js

```diff
@@ -270,6 +270,9 @@ function showDesktopNotification() {
         iconURL: './media/icons/icon-32_active.png',
         onClick: toggleSidebar
     });
+
+    // note that we have shown our notification so we don't try showing it again
+    simpleStorage.shownNotification = true;
```

Comment:
We need to note when we've shown a notification so that we don't start 60 second timers every time 24 hours has passed and a new session starts